Add template section "CSS & JS"

yurabakhtin opened the pull request:
https://github.com/humhub/custom-pages-internal/issues/86

yurabakhtin commented:
> You're still working on Tests?

@luke- Yes, I should fix them.

luke- commented:
@yurabakhtin One question, is Update/Enable called when manually update the source (e.g. git pull) of the module? Is there a possibility to trigger it manually?

yurabakhtin commented:
> One question, is Update/Enable called when manually update the source (e.g. git pull) of the module?

@luke- No, now after I have deleted the line it is not called on run migration manually from command line or from administration.

I have deleted it from the migration to avoid similar errors when db tables will be changed, otherwise we should always support new db structure in the migration `m250422_145411_default_template` as I firstly did [here](https://github.com/humhub/custom-pages/pull/440/commits/234f2edbfda697f81f4588ca5e631b6415e3a1e4), but then deleted this solution because it was not fast to understand why tests are broken.

I can try to call the line on the event `MigrationService::EVENT_AFTER_MIGRATION` but I am not sure it is a good idea because it will be called after each new migration, however now we call it after each module update so maybe it is ok, what do you think?

> Is there a possibility to trigger it manually?

Yes, we have the command:

```
- custom-pages
    custom-pages/refresh-default-templates        Refresh default templates from all active modules
```
